types/next/head.d.ts

```diff
@@ -1,8 +1,8 @@
 import * as React from "react";
 
 export function defaultHead(): JSX.Element[];
-export default class extends React.Component {
+export default class Head extends React.Component {
     static canUseDOM: boolean;
-    static peek(): Array<React.ReactElement<any>>;
-    static rewind(): Array<React.ReactElement<any>>;
+    static peek(): React.ReactElement<any>[];
+    static rewind(): React.ReactElement<any>[];
```

Comment:
Will switching these to `React.ReactNodeArray` still work?

Reply:
Just double checked. This is the result of calls to `React.cloneElement` which always returns "sub-classes" of `React.ReactElement` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L213). So I think this is correct. I reverted this change and kept the `Array<>` version for non-standard types as per the lint rules.